From 16cf7fc2ee9ca0ad52b12e30617522a533022384 Mon Sep 17 00:00:00 2001
From: Philippe Gerum <rpm@xenomai.org>
Date: Thu, 12 Jul 2018 17:15:19 +0200
Subject: [PATCH 104/137] x86: paravirt: disable if CONFIG_IPIPE

Some changes introduced by the basic interrupt pipeline code for x86
break the paravirtualization code. Disable the latter.
---
 arch/x86/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index acaf363..51b15c3 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -743,6 +743,7 @@ if HYPERVISOR_GUEST
 
 config PARAVIRT
 	bool "Enable paravirtualization code"
+	depends on !IPIPE
 	---help---
 	  This changes the kernel so it can modify itself when it is run
 	  under a hypervisor, potentially improving performance significantly
-- 
1.9.1

